lib/eventboss/long_poller.rb

```diff
@@ -53,6 +53,9 @@ def run
       @launcher.poller_stopped(self)
     rescue Eventboss::Shutdown
       @launcher.poller_stopped(self)
+    rescue Aws::SQS::Errors::NonExistentQueue
+      handle_exception(exception, poller_id: id)
+      @launcher.poller_stopped(self)
```

Comment:
Will it go to airbrake notifier if it's enabled? It would be nice if it does.

Reply:
yes, that's exactly why I updated it